src/main/java/com/bandwidth/sdk/examples/CredentialsExamples.java

```diff
@@ -125,9 +125,9 @@ public static void main(final String[]args) {
         }
 
         // Finally just set them directly
-        userId = "your User Id here";
-        apiToken = "your API Token here";
-        apiSecret = "your API Secret here";
```

Comment:
Ensure that if user provides any credential with white spaces, it will be removed on SDK.